### PR TITLE
Use equivalent addon-sdk apis instead of raw chrome interfaces

### PR DESCRIPTION
**Pull request by cvializ**
@andrewconner we're down to 0 warnings about low-level api usage

### CODE REVIEW

File: packrat/adapters/firefox/data/scripts/workers/sound_worker.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+function log() {
```
Comment (andrewconner) [inline packrat/adapters/firefox/data/scripts/workers/sound_worker.js:1]: Firefox API is so weird. Nice :)

**Comment by andrewconner** [comment]
Slick

**Comment by cvializ** [comment]
Feeling good about this one. I'll merge, and we should test it aggressively
- "Look here" flying images
- Notification bell
- deep linking

**Comment by andrewconner** [comment]
:+1:

**Pull request merged**